src/main/resources/archetype-resources/deployment/k8s/branch/all.jsonnet

```diff
@@ -0,0 +1,180 @@
+local vars = std.extVar('rebuyTemplateVariables');
+
+local presets = {
+  production: {
+    replicas: 2,
+    liveTraffic: true,
+    cpuRequests: '100m',
+  },
+  testing: {
+    replicas: 1,
+    liveTraffic: false,
+    cpuRequests: '100m',
+  },
+  staging: {
+    replicas: 2,
+    liveTraffic: true,
+    cpuRequests: '100m',
+  },
+};
+
+local preset = presets[vars.presetName];
+
+local metadata = {
+  name: vars.applicationName,
+  labels: {
+    name: vars.applicationName,
+    'app.kubernetes.io/name': vars.projectName,
+    'app.kubernetes.io/version': vars.release,
+    'rebuy.com/release': vars.release,
+    'rebuy.com/preset': vars.presetName,
+    'rebuy.com/live-traffic': if preset.liveTraffic then 'enabled' else 'disabled',
+    'rebuy.com/toolstack': 'jvm.spring-boot.v2.6',
+    team: '${team}',
```

Comment:
That's deprecated. I'll create a ticket.

Reply:
https://rebuyer.atlassian.net/browse/CLOUD-3045

Reply:
Thanks https://github.com/rebuy-de/rebuy-silo-archetype/pull/172/commits/83b28216be376056afdb70c903ad529f0c619f7f